genrl/core/buffers.py

```diff
@@ -3,7 +3,9 @@
 from typing import NamedTuple, Tuple
 
 import numpy as np
+import reverb
 import torch
+import tensorflow as tf
```

Comment:
Do we need to add tensorflow as a requirement?

Reply:
Yeah. `dm-reverb` depends on tensorflow

Reply:
Hmm okay